ruby-monday-blog/db/migrate/20150818035637_add_name_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddNameToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :first_name, :string
+    add_column :users, :last_name, :string
```

Comment:
I'm not really sure how to do a conditional NOT NULL constraint using `ActiveRecord::Migration` to enforce `null: false` only where the role is `:author`.  I think this would have to be a manual SQL execute... this article looks promising: http://stackoverflow.com/questions/10273750/sql-conditional-not-null-constraint

@matugm could use your help, here!  Need a SQL pro :)